mbx/s2-4-suppprobs.mbx

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<section xml:id="sec_induction-suppprobs" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
oops! I copied from Ch. 4 Supplementary and left the xml:id.  

Anyway I tried to push the fixed ch2-supp but got this and now I'm stuck:

% git push origin ch2-supp
Username for 'https://github.com': kentemorrison
Password for 'https://kentemorrison@github.com': 
To https://github.com/kentemorrison/ibl-combinatorics.git
 ! [rejected]        ch2-supp -> ch2-supp (non-fast-forward)
error: failed to push some refs to 'https://github.com/kentemorrison/ibl-combinatorics.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.